src/UserConfig.elm

```diff
@@ -1651,6 +1651,411 @@ Podemos cerrar su cuenta de inmediato si pensamos que ha:
     }
 
 
+thaiTexts : Texts
+thaiTexts =
+    { addEvent = "เพิ่มกิจกรรม"
+    , addressTooLong = \length maxLength -> "ที่อยู่ยาว " ++ String.fromInt length ++ " ตัวอักษร กรุณาตัดให้สั้นลงอยู่ใน " ++ String.fromInt maxLength ++ " ตัวอักษร"
+    , addressTooShort = \length minLength -> "ที่อยู่สั้น " ++ String.fromInt length ++ " ตัวอักษร ต้องมีอย่างน้อย " ++ String.fromInt minLength ++ " ตัวอักษร"
+    , aLoginEmailHasBeenSentTo = "อีเมลล็อกอินถูกส่งไปที่ "
+    , anAccountDeletionEmailHasBeenSentTo = "อีเมลล์สำหรับลบบัญชีถูกส่งไปที่ "
+    , andNanonymousNattendees =
+        \attendeeCount ->
+            if attendeeCount == 1 then
+                "และมีผู้เข้าร่วมที่ไม่ระบุชื่อ\nอีกหนึ่งคน"
+
+            else
+                "และมีผู้เข้าร่วมที่ไม่ระบุชื่อ\nอีก " ++ String.fromInt attendeeCount ++ " คน"
+    , andOneNanonymousNattendee = "และมีผู้เข้าร่วมที่ไม่ระบุชื่อ\nอีกหนึ่งคน"
+    , aPlaceToJoinGroupsOfPeopleWithSharedInterests = "สถานที่ที่คุณสามารถเข้าร่วมกับกลุ่มคนที่มีความสนใจเดียวกัน"
+    , beginsIn = "เริ่มในอีก "
+    , belowNCharactersPlease = \n -> "ต้องมีตัวอักษรน้อยกว่า " ++ String.fromInt n ++ " ตัวอักษร"
+    , buttonOnAGroupPage = "\" ปุ่มในหน้ากลุ่ม"
+    , byContinuingYouAgreeToThe = "โดยการดำเนินการต่อคุณยอมรับ "
+    , cancel = "ยกเลิก"
+    , cancelChanges = "ยกเลิกการเปลี่ยนแปลง"
+    , cancelEvent = "ยกเลิกกิจกรรม"
+    , checkYourSpamFolderIfYouDonTSeeIt = "กรุณาตรวจสอบโฟลเดอร์สแปมหากคุณไม่เห็น"
+    , chooseWhatTypeOfEventThisIs = "เลือกประเภทของกิจกรรมนี้"
+    , codeOfConduct = "ประเด็นสำคัญในการปฏิบัติ"
+    , codeOfConduct1 = "นี่คือคำแนะนำเพื่อให้คุณปฏิบัติตามกฎ \"อย่าทำตัวเป็นคนไม่ดี\":"
+    , codeOfConduct2 = "• ให้ความเคารพกับผู้คนไม่ว่าจะเป็นเรื่องเชื้อชาติ, เพศ, ตัวตนทางเพศ, สัญชาติ, รูปลักษณ์, หรือลักษณะอื่นๆ ที่เกี่ยวข้อง."
+    , codeOfConduct3 = "• ให้ความเคารพกับผู้จัดกลุ่ม พวกเขาใช้เวลาในการจัดงานและพวกเขายินดีที่จะเชิญคนที่ไม่รู้จัก เด็ดขาดอย่าทำลายความไว้วางใจของพวกเขาในคุณ!"
+    , codeOfConduct4 = "• สำหรับผู้จัดกลุ่ม: ทำให้คนรู้สึกว่าพวกเขาเป็นส่วนหนึ่งของกลุ่ม คนจะไม่สามารถเข้าร่วมได้ถ้าพวกเขารู้สึกว่าตัวเองเป็นคนนอก."
+    , codeOfConduct5 = "• ถ้ามีคนทำตัวเป็นคนไม่ดี คุณไม่ต้องทำตัวเป็นคนไม่ดีกลับ ขอให้พวกเขาหยุด และถ้าสิ่งนั้นไม่ทำงาน หลีกเลี่ยงพวกเขาและอธิบายปัญหาที่นี่ "
+    , copyPreviousEvent = "คัดลอกกิจกรรมก่อนหน้านี้"
+    , createEvent = "สร้างกิจกรรม"
+    , createGroup = "สร้างกลุ่ม"
+    , creatingOne = "การสร้าง"
+    , creditGoesTo = " ความยินดีควรไปยัง "
+    , dateValueMissing = "ขาดค่าวันที่"
+    , daysUntilEvent = \days -> "วันจนถึงกิจกรรม: " ++ String.fromInt days
+    , deleteAccount = "ลบบัญชี"
+    , deleteGroup = "ลบกลุ่ม"
+    , describeWhatYourGroupIsAboutYouCanFillOutThisLater = "อธิบายว่ากลุ่มของคุณเกี่ยวกับอะไร (คุณสามารถกรอกนี้ได้ภายหลัง)"
```

Comment:
คุณสามารถกรอกนี้ได้ภายหลัง should be reworded

Reply:
@CharlonTank I guess I should wait with approving the merge until this is fixed?

Reply:
Oh I didn't see that one, yes, we should wait until I get verified by some Thai speaking people :)

Reply:
_Key: `describeWhatYourGroupIsAboutYouCanFillOutThisLater`_

**Original**: Describe what your group is about (you can fill out this later)

**Old Translation**: อธิบายว่ากลุ่มของคุณเกี่ยวกับอะไร (คุณสามารถกรอกนี้ได้ภายหลัง)

_Dtinth Review: "คุณสามารถกรอกนี้ได้ภายหลัง" should be reworded_

**New Translation**: อธิบายว่ากลุ่มของคุณเกี่ยวกับอะไร (คุณสามารถเติมข้อมูลนี้ภายหลัง)

**Why**: "เติมข้อมูล" is a more natural way to express "fill out" in Thai.

Reply:
```suggestion
    , describeWhatYourGroupIsAboutYouCanFillOutThisLater = "อธิบายว่ากลุ่มของคุณเกี่ยวกับอะไร (คุณสามารถเติมข้อมูลนี้ภายหลัง)"
```